db/migrate/20210426193009_move_approved_queued_posts_topic_and_post_data.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class MoveApprovedQueuedPostsTopicAndPostData < ActiveRecord::Migration[6.1]
+  def up
+    DB.query_single <<~SQL
+      UPDATE reviewables r
+      SET topic_id = (payload->>'created_topic_id')::int, target_id = (payload->>'created_post_id')::int
+      WHERE r.type = 'ReviewableQueuedPost' AND r.status = #{Reviewable.statuses[:approved]}
+      AND (r.payload->>'created_topic_id') IS NOT NULL AND (r.payload->>'created_post_id') IS NOT NULL
+    SQL
+  end
+
+  def down
```

Comment:
I would just make this non reversible. no need to have a down migration here. It is not really that reversible anyway, model will create new things afterwards.